CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+## 0.11.0 (July 15, 2021)
+
+#### :hourglass: No Longer Supported
+
+- Remove Python 2.7 support
+
+#### :bug: Bug Fix
+
+- Fix issue where `wandb.watch()` broke model saving in pytorch
+- Fix issue where uniform sweep parameters were parsed as int_uniform
+- Fix issue where file_stream thread was killed on 4xx errors
+
+#### :nail_care: Enhancement
+
+- Improve performance of artifact logging by making it non-blocking
+- Add wandb integration for Stable-benchmarks-3
```

Comment:
should be `Stable-Baselines3`, no?

Reply:
oops apologies @araffin
Fixed in https://github.com/wandb/client/pull/2402

Reply:
oops !